### PR TITLE
Describe the PieChart - annotations demo

### PR DESCRIPTION
**Pull request by AlexJameson**
http://jsserver:81/Demos-20.2/WidgetsGallery/Demo/Charts/PieWithAnnotations/jQuery/Light/

### CODE REVIEW

File: JSDemos/Demos/Charts/PieWithAnnotations/description.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Annotations are containers for images, text blocks, and custom content that provide additional information about the visualized data.
+
+To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
+
+Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/#type) attribute should be set to *"text"*, *"image"* or *"custom"*. In this demo, the type of all annotations is *"image"*.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/PieWithAnnotations/description.md]:
```suggestion
Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/#type) attribute should be set to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"image"*.
```

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/Charts/Annotation/description.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Annotations are images and text blocks that provide additional information on the visualized data.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/Annotation/description.md]: Just another version... Again, the reason is the word "provide". 

```suggestion
Annotations are images and text blocks that add information to the visualized data.
```

File: JSDemos/Demos/Charts/Annotation/description.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 
-To configure annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) option. Each object configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over these settings.
+To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) option. Each object configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over these settings.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/Annotation/description.md]: I made a change to a similar block in another merge request. I don't know why you add the word "individual" if you then switch to an "array". Then the next sentence talks about "individual" annotations again (repetition). 

Here's another option: 

```suggestion
To create annotations, populate the widget's [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) array. Each object in the array configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
```

File: JSDemos/Demos/Charts/Annotation/description.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 
 Each annotation should have the [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type) set to *"text"* or *"image"*. Depending on the **type**, specify the [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#text) or [image](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/image/) option. You can also add a [description](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#description) that will be displayed in a tooltip.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/Annotation/description.md]: Please see my other comments about this part (the word "should").

File: JSDemos/Demos/Charts/PieWithAnnotations/description.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Annotations are containers for images, text blocks, and custom content that provide additional information about the visualized data.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/PieWithAnnotations/description.md]: Similar comment as above or in other merge request ("provide"). A lot of this text repeats what I already commented on - can you please review those comments and apply changes. Thanks.

File: JSDemos/Demos/Charts/Annotation/description.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
+You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type) attribute to *"text"*, *"image"*, or *"custom"*. Depending on the **type**, specify the [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#text) or [image](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/image/) option. You can also add a [description](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#description) that will be displayed in a tooltip.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Annotation/description.md]:
```suggestion
You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type) option to *"text"*, *"image"*, or *"custom"*. Depending on the **type**, specify the [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#text) or [image](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/image/) option. You can also add a [description](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#description) that will be displayed in a tooltip.
```

File: JSDemos/Demos/Charts/PieWithAnnotations/description.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Annotations are containers for images, text blocks, and custom content that display additional information about the visualized data.
+
+To create annotations, populate the widget's [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/) array. Each object in the array configures an individual annotation. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
+
+You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/#type) attribute to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"image"*.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/PieWithAnnotations/description.md]:
```suggestion
You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/annotations/#type) option to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"image"*.
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**